### PR TITLE
test: pullDividendPayment test suite cover at 100%

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: src/procedures/__tests__/PullDividendPayment.ts [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+          shareholders: [{ address: '0x01', paymentReceived: false, excluded: false }],
+        })
+      );
+
+      // Real call
+      await expect(target.prepareTransactions()).rejects.toThrowError(
+        new PolymathError({
+          code: ErrorCode.ProcedureValidationError,
+          message:
+            'Current wallet 0x04 cannot receive dividend payments. Reason: not a shareholder',
```
Comment (victorwiebe) [inline src/procedures/__tests__/PullDividendPayment.ts]: You could make 0x04 a variable to reuse it here in this text

File: src/procedures/__tests__/PullDividendPayment.ts [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+          shareholders: [{ address: '0x01', paymentReceived: true, excluded: false }],
+        })
+      );
+
+      // Real call
+      await expect(target.prepareTransactions()).rejects.toThrowError(
+        new PolymathError({
+          code: ErrorCode.ProcedureValidationError,
+          message:
+            'Current wallet 0x01 cannot receive dividend payments. Reason: already received payment',
```
Comment (victorwiebe) [inline src/procedures/__tests__/PullDividendPayment.ts]: You could make 0x01 a variable to reuse it here in this text

File: src/procedures/__tests__/PullDividendPayment.ts [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+          shareholders: [{ address: '0x01', paymentReceived: false, excluded: true }],
+        })
+      );
+
+      // Real call
+      await expect(target.prepareTransactions()).rejects.toThrowError(
+        new PolymathError({
+          code: ErrorCode.ProcedureValidationError,
+          message:
+            'Current wallet 0x01 cannot receive dividend payments. Reason: address belongs to exclusion list',
```
Comment (victorwiebe) [inline src/procedures/__tests__/PullDividendPayment.ts]: You could make 0x01 a variable to reuse it here in this text

File: src/procedures/__tests__/PullDividendPayment.ts [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+        })
+      );
+    });
+
+    test('should add a transaction to pull a dividend payment', async () => {
```
Comment (monitz87) [inline src/procedures/__tests__/PullDividendPayment.ts]: "... to pull dividend payments"

**Pull request merged**